lib/llvm-backend/src/code.rs

```diff
@@ -2693,85 +2693,379 @@ impl FunctionCodeGenerator<CodegenError> for LLVMFunctionCodeGenerator {
             }
             Operator::F32Min => {
                 let (v1, v2) = state.pop2()?;
-                let res = builder
-                    .build_call(intrinsics.minimum_f32, &[v1, v2], &state.var_name())
-                    .try_as_basic_value()
-                    .left()
-                    .unwrap();
```

Comment:
Can you add comments in each of this implementations adding context, such as:

```
LLVM minmax doesn't work on the versions I tried (8, 9) because ...
So we are writing our own implementation of minmax each time we call `f32min` ...
```

Reply:
Done.